package.json

```diff
@@ -110,7 +110,6 @@
         "karma-jasmine": "~5.1.0",
         "karma-jasmine-html-reporter": "~2.0.0",
         "ng-packagr": "^14.1.0",
-        "@nrwl/devkit": "// remove this when <https://github.com/nrwl/nx/issues/11653> is resolved",
```

Comment:
No, you would need to remove `@nrwl/devkit` completely and not just the comment :smile:

Reply:
npm will remove the dup entry every time. i did not reverse that.